hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -140,6 +140,8 @@ private boolean shouldUseSimpleFees(FeeContext feeContext) {
                     CONSENSUS_SUBMIT_MESSAGE,
                     CONSENSUS_UPDATE_TOPIC,
                     CONSENSUS_DELETE_TOPIC,
+                    // TODO: add base fee for SCHEDULE_CREATE_CONTRACT_CALL
```

Comment:
May be we dont need this here unless we change to depend on different enum than HedearFunctionality